stan/math/prim/prob/gaussian_dlm_obs_lpdf.hpp

```diff
@@ -136,6 +136,8 @@ inline return_type_t<T_y, T_F, T_G, T_V, T_W, T_m0, T_C0> gaussian_dlm_obs_lpdf(
       f = multiply(transpose(F_ref), a);
       // Q_t = F'_t R_t F_t + V_t
       Q = quad_form_sym(R, F_ref) + V_ref;
+      if (i == 0)
+        check_pos_definite(function, "Q", Q);
```

Comment:
I don't think we need a `check_pos_definite(function, "Q", Q);` here because the very next line will throw if `Q` is not positive definite.

Reply:
Good point. Fixed